libsvm-kitware/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+cd TPL/libsvm-3.1-custom
+
+make lib
+mkdir $PREFIX/lib
+mv libsvm* $PREFIX/lib/
```

Comment:
My bad - I'm not sure how I missed these.  The command failed, and I went looking for Python folders, found the one in root, and did my find command in there.  Whoops.  I found it.

Now Conda won't package this because it's trying to compile these files to .pyc files and failing somehow, but that's fine - I can work on that.  Just know that this recipe won't work right now.